target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -1,5 +1,17 @@
 include ./common-tp-link.mk
 
+define Device/mercury_mw4530r-v1
+  $(Device/tplink-8mlzma)
+  SOC := ar9344
+  DEVICE_VENDOR := Mercury
+  DEVICE_MODEL := MW4530R
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ledtrig-usbport
+  TPLINK_HWID := 0x45300001
+  SUPPORTED_DEVICES += tl-wdr4300
+endef
+TARGET_DEVICES += mercury_mw4530r-v1
```

Comment:
Since this is only a TP-Link clone, but not a TP-Link device (as far as I understand it), I'd consider moving it to generic.mk